.github/workflows/workflows.json

```diff
@@ -29,6 +29,7 @@
   "datalabeling",
   "datastore/functions",
   "datacatalog/quickstart",
+  "document-ai",
```

Comment:
**Praise:** Thanks for redo-ing this PR and removing the `document-ai/snippets/` sub-folder.